src/taskdb/mod.rs

```diff
@@ -16,21 +16,19 @@ mod working_set;
 /// A TaskDb is the backend for a replica.  It manages the storage, operations, synchronization,
 /// and so on, and all the invariants that come with it.  It leaves the meaning of particular task
 /// properties to the replica and task implementations.
-pub(crate) struct TaskDb {
-    storage: Box<dyn Storage>,
-}
+pub(crate) struct TaskDb;
```

Comment:
This type is kind of unnecessary now, but we can factor it away in a followup, just to limit the number of changes in this PR.